# left mbot arm velocity controller params

left_arm_joint0_velocity_controller:
  type: effort_controllers/JointVelocityController
  joint: left_arm_joint0
  pid: {p: 10.0, i: 0.0, d: 0.0, i_clamp: 0.0}

left_arm_joint1_velocity_controller:
  type: effort_controllers/JointVelocityController
  joint: left_arm_joint1
  pid: {p: 10.0, i: 0.0, d: 0.0, i_clamp: 0.0}

left_arm_joint2_velocity_controller:
  type: effort_controllers/JointVelocityController
  joint: left_arm_joint2
  pid: {p: 10.0, i: 0.0, d: 0.0, i_clamp: 0.0}

left_arm_joint3_velocity_controller:
  type: effort_controllers/JointVelocityController
  joint: left_arm_joint3
  pid: {p: 10.0, i: 0.0, d: 0.0, i_clamp: 0.0}

left_arm_joint4_velocity_controller:
  type: effort_controllers/JointVelocityController
  joint: left_arm_joint4
  pid: {p: 10.0, i: 0.0, d: 0.0, i_clamp: 0.0}

left_arm_joint5_velocity_controller:
  type: effort_controllers/JointVelocityController
  joint: left_arm_joint5
  pid: {p: 10.0, i: 0.0, d: 0.0, i_clamp: 0.0}

left_arm_joint6_velocity_controller:
  type: effort_controllers/JointVelocityController
  joint: left_arm_joint6
  pid: {p: 10.0, i: 0.0, d: 0.0, i_clamp: 0.0}
